lib/widgets/components/card.dart

```diff
@@ -58,15 +58,22 @@ abstract class CardBase extends StatelessWidget {
     );
   }
 
+  void Function()? _onTap(BuildContext context) {
+    if (onTap == null) return null;
+    return () {
+      onTap!(context);
+    };
+  }
+
   @override
   Widget build(BuildContext context) {
     return Tappable(
-      elevation: 1,
+      elevation: disabled ? 0 : 1,
       padding: EdgeInsets.all(dense ? 16 : 24),
       color: color,
       borderRadius: BorderRadius.circular(24),
       borderColor: borderColor,
-      onTap: onTap,
+      onTap: _onTap(context),
```

Comment:
Is there any reason why a card can be disabled, and have an onTap function?

Reply:
An oversight. Fixed